contents/docs/tutorials/beyond-the-user-model.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Going Beyond the User Model
```

Comment:
I get why you picked this, but I found it confusing.

Maybe "Tracking how teams use your product"?